include/wx/osx/bitmap.h

```diff
@@ -64,20 +62,36 @@ class WXDLLIMPEXP_CORE wxMask: public wxObject
 
     // a 8 bit depth mask
     void* GetRawAccess() const;
-    int GetBytesPerRow() const { return m_bytesPerRow ; }
+    int GetBytesPerRow() const;
```

Comment:
I think those should have `OSX` prefix or be declared in `wxMaskBase` (and implemented in the existing `wxMask` implementations deriving from it) to make it more clear that they're not part of the public API or, alternatively, to make them part of the public API.

Otherwise it will be too easy to write code which compiles with wxMac, but not any other ports.